escalate/core/views/model_tag.py

```diff
@@ -33,7 +33,7 @@ def post(self, request, *args, **kwargs):
             #above url not being redirected to in super.post so I brute-forced redirect
             return redirect(f'/{model_name}/{model_pk}')
```

Comment:
I tried changing the success url if the cancel button was clicked, but when I tested it, it doesn't work, so I improvised for now. Hoping to see if you had any ideas on why it didn't work.